hack/install-latest.yaml

```diff
@@ -164,6 +164,40 @@ status:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
```

Comment:
IIRC, `hack/install-latest.yaml` should be checked in by release build or while cutting the release manually.

Reply:
This is changed to avoid CI fail. It seems not harmful to set a new CRD. 
Did you mean we need to change current CI not fail for potential case?

Reply:
after sync-up with @shashidharatd, we agree to keep this change. We will have discussion later to see if install-latest.yaml is only for release but not in code.

Reply:
`install-latest.yaml` has nothing to do with release and needs be maintained for any PR that would affect its contents.